console/src/components/Common/GqlCompatibilityWarning/GqlCompatibilityWarning.tsx

```diff
@@ -4,7 +4,12 @@ import WarningSymbol from '../WarningSymbol/WarningSymbol';
 
 const gqlPattern = /^[_A-Za-z][_0-9A-Za-z]*$/;
 
-const GqlCompatibilityWarning = ({ identifier, className = null }) => {
+export interface GqlCompatibilityWarningProps {
+  identifier: string;
+  className?: string;
+}
+
+const GqlCompatibilityWarning: React.FC<GqlCompatibilityWarningProps> = ({ identifier, className }) => {
```

Comment:
Let's add `className = ''`. Or `className || ''` in line 25.
This way we won't end up with `undefined` as a class name 😊

Reply:
Thanks for the review. Indeed. It's better